recipes/tensorflow-lattice/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "tensorflow-lattice" %}
+{% set version = "2.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tensorflow_lattice-{{ version }}.tar.gz
+  sha256: 68ba98a98e8f73de1c1f63d31e40a1308b53214c860fe96d39a28b485e9adaeb
+  patches:
+    # https://github.com/tensorflow/lattice/pull/68
+    - 0001-setup.py-fix-sklearn-scikit-learn.patch
+
+build:
+  number: 0
+  # There is not yet any conda-forge build for Windows or Python 3.9.
+  skip: true  # [win or py>=39]
```

This looks like it could be `noarch` and then it would automatically work when we release a Tensorflow Py39 build.
```suggestion
  noarch: python
```